### ISSUE TITLE
Filters not working

### ISSUE THREAD

**Issue by mitexleo** [BDX-town/Mangane#336]
Even I've set it to "Drop instead of hide" posts are still appearing in my feed. 

![SmartSelect_20240706-125859_Vivaldi](https://github.com/BDX-town/Mangane/assets/114842102/e2c2f617-8563-4149-86f1-e0a611572aaa)
![SmartSelect_20240706-130023_Vivaldi](https://github.com/BDX-town/Mangane/assets/114842102/c3ae28ba-956b-439f-a159-98193d67acbe)

**Comment by Cl0v1s** [comment]
Hi ! Do you have multiple filters applied ? 

When you choose "drop instead of hide" option, status are filtered by Akkoma and not by the frontend

**Comment by mitexleo** [comment]
Yeah, I do have multiple filters applied. 






---- On Sun, 07 Jul 2024 21:41:48 +0600 ***@***.*** wrote ----



Hi ! Do you have multiple filters applied ?

When you choose "drop instead of hide" option, status are filtered by Akkoma and not by the frontend

**Comment by Cl0v1s** [comment]
Are you sure that the hidden post you shared is related to the filter with the drop setting enabled ?

**Comment by mitexleo** [comment]
> Are you sure that the hidden post you shared is related to the filter with the drop setting enabled ?

Yeah. For testing I removed all filters and added "gop". The behavior is still same. Posts are still appearing as hidden. 
![SmartSelect_20240707-224334_Vivaldi](https://github.com/BDX-town/Mangane/assets/114842102/397ecb82-c03a-47e8-a385-1a7e8c197247)
![Screenshot_20240707-224403_Vivaldi](https://github.com/BDX-town/Mangane/assets/114842102/84daa633-7483-407d-92ec-55a527ba5c56)
![Screenshot_20240707-224409_Vivaldi](https://github.com/BDX-town/Mangane/assets/114842102/e431923f-f456-4418-8ad4-b718d151c65b)

**Comment by mitexleo** [comment]
> > Are you sure that the hidden post you shared is related to the filter with the drop setting enabled ?
> 
> Yeah. For testing I removed all filters and added "gop". The behavior is still same. Posts are still appearing as hidden. ![SmartSelect_20240707-224334_Vivaldi](https://private-user-images.githubusercontent.com/114842102/346350591-397ecb82-c03a-47e8-a385-1a7e8c197247.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAzNzExMjUsIm5iZiI6MTcyMDM3MDgyNSwicGF0aCI6Ii8xMTQ4NDIxMDIvMzQ2MzUwNTkxLTM5N2VjYjgyLWMwM2EtNDdlOC1hMzg1LTFhN2U4YzE5NzI0Ny5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwN1QxNjQ3MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02OTRkMzhjMmE1OWY3YzFlNjk5MDVhM2U3MmQyNDY2ZGNjOTc0MDZiYWZjY2Y5OGQwNmMxY2M0NDMwYjRiM2I4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.QvyP8jEYWNOXFunNbHQQW4rvjFqiB1TMl6LdNw5gCAs) ![Screenshot_20240707-224403_Vivaldi](https://private-user-images.githubusercontent.com/114842102/346350611-84daa633-7483-407d-92ec-55a527ba5c56.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAzNzExMjUsIm5iZiI6MTcyMDM3MDgyNSwicGF0aCI6Ii8xMTQ4NDIxMDIvMzQ2MzUwNjExLTg0ZGFhNjMzLTc0ODMtNDA3ZC05MmVjLTU1YTUyN2JhNWM1Ni5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwN1QxNjQ3MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYWExOGUwODg3OWJlYTE5ZGVkN2YxMjU0MDlmNDQzMDVmNTdmYmMwNGQ0M2UzMWM2YmFiOTA3MTY3MDExMWM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ocQlNHElpZkRWKluRorqOgT0m_rxSoxwXXVrey1IF0k) ![Screenshot_20240707-224409_Vivaldi](https://private-user-images.githubusercontent.com/114842102/346350620-e431923f-f456-4418-8ad4-b718d151c65b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAzNzExMjUsIm5iZiI6MTcyMDM3MDgyNSwicGF0aCI6Ii8xMTQ4NDIxMDIvMzQ2MzUwNjIwLWU0MzE5MjNmLWY0NTYtNDQxOC04YWQ0LWI3MThkMTUxYzY1Yi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwN1QxNjQ3MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZWZmMjc2MGZkZjljMTczMDk1Njg0NTE2MWRkNWMxNDBkYmEwZDk3YjRkZDk3MzRhMGQwZjY5MGFkM2ZjNjhkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.x1Xl1dKUSNNZTa0Zvp3l-8B4ACkJZbokEDZML3ZqZdI)

It's same across all feeds.

**Comment by Cl0v1s** [comment]
Thanks for further testing ! 
Can you confirm you are using Akkoma as a backend ? Is that's the case, can you tell me which Akkoma version you are using ? :)

**Comment by mitexleo** [comment]
I'm using Akkoma v3.13.1-0-gc02e343 ..


Thanks for looking into the issue ❤






---- On Wed, 10 Jul 2024 01:45:31 +0600 ***@***.*** wrote ----



Thanks for further testing !
Can you confirm you are using Akkoma as a backend ? Is that's the case, can you tell me which Akkoma version you are using ? :)

**Comment by Cl0v1s** [comment]
This is definitely an issue on Akkoma's side, as the drop setting is enabling a filter on the server-side
I won't be able to help there... :/
